lib/kubernetes-deploy/kubectl.rb

```diff
@@ -9,6 +9,7 @@ class ResourceNotFoundError < StandardError; end
 
     def initialize(namespace:, context:, logger:, log_failure_by_default:, default_timeout: DEFAULT_TIMEOUT,
       output_is_sensitive_default: false)
+      @kubeconfig = KubeclientBuilder.new.kubeconfig
```

Comment:
Should `kubeconfig` be a class method on `KubeclientBuilder`? I can't decide.

Reply:
If we're not doing an integration test with `DeployTask`, there's no reason for it not to be. Let me know what you choose.

Reply:
Since it doesn't seem to be impacted by the state of any given `KubeclientBuilder` object, I'm inclined to make it a class method

Reply:
Cool. I'm putting the energy I have available into https://github.com/Shopify/kubernetes-deploy/pull/439 because that's the one that is useful to me. Once that is merged, I'll look at these other PRs again.